Emby.Server.Implementations/Channels/ChannelDynamicMediaSourceProvider.cs

```diff
@@ -27,12 +28,9 @@ public ChannelDynamicMediaSourceProvider(IChannelManager channelManager)
         /// <inheritdoc />
         public Task<IEnumerable<MediaSourceInfo>> GetMediaSources(BaseItem item, CancellationToken cancellationToken)
         {
-            if (item.SourceType == SourceType.Channel)
-            {
-                return _channelManager.GetDynamicMediaSources(item, cancellationToken);
-            }
-
-            return Task.FromResult<IEnumerable<MediaSourceInfo>>(new List<MediaSourceInfo>());
+            return item.SourceType == SourceType.Channel
+                ? _channelManager.GetDynamicMediaSources(item, cancellationToken)
+                : Task.FromResult<IEnumerable<MediaSourceInfo>>(new List<MediaSourceInfo>());
```

Comment:
```suggestion
                : Task.FromResult<IEnumerable<MediaSourceInfo>>(Enumerable.Empty<MediaSourceInfo>());
```
You'll have to add `using System.Linq;` at the top of the file